routers/user/home.go

```
@@ -332,11 +332,18 @@ func Issues(ctx *context.Context) {
 	}
 
 	var commitStatus = make(map[int64]*models.CommitStatus, len(issues))
-	for _, issue := range issues {
-		issue.Repo = showReposMap[issue.RepoID]
-
-		if isPullList {
-			commitStatus[issue.PullRequest.ID], _ = issue.PullRequest.GetLastCommitStatus()
+	for i := 0; i < len(issues); {
+		issue := issues[i]
+		repo, ok := showReposMap[issue.RepoID]
+		if ok {
+			issue.Repo = repo
+
+			if isPullList {
+				commitStatus[issue.PullRequest.ID], _ = issue.PullRequest.GetLastCommitStatus()
```

Is it certain that the error can be ignored?